src/subscription_manager/intentstore_interface.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function, division, absolute_import
+
+#
+# Copyright (c) 2018 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public License,
+# version 2 (GPLv2). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+# along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+#
+# Red Hat trademarks are not licensed under GPLv2. No permission is
+# granted to use or replicate Red Hat trademarks that are incorporated
+# in this software or its documentation.
+#
+
+
+def set_intent_service_level(service_level):
+    try:
+        import intentctl.intentfiles
+        store = intentctl.intentfiles.IntentStore.read(intentctl.intentfiles.USER_INTENT)
+    except ImportError:
+        print("WARNING: Could not import from module intentctl. Intent SLA value not saved locally.")
+    else:
+        # if empty, set it to null
+        if service_level is None or service_level == "":
+            service_level = None
+
+        store.set("service_level_agreement", service_level)
+        store.write()
+        print("Intent SLA value successfully saved locally.")
```

Comment:
Is it really necessary to print this information to `stdout`? I guess that writing this information to `/var/log/rhsm/rhsm.log` would be enough. Example:

```python
import logging
log = logging.getLogger(__name__)

def some_function():
    log.info("Intent SLA value successfully saved locally.")
```